OasisDEX/oasis-borrow
# Deploy new fee tier for G-UNI and update contract address

# [Deploy new fee tier for GUNI - and update contract addresses to use it](https://app.shortcut.com/oazo-apps/story/6168/deploy-new-fee-tier-for-guni-and-update-contract-addresses-to-use-it)

OB Cache PR -> https://github.com/OasisDEX/oasis-borrow-cache/pull/152

New contract on mainnet -> https://etherscan.io/address/0xf22f17b1d2354b4f4f52e4d164e4eb5e1f0a6ba6#code
  
## Changes 👷‍♀️
- Deploy new version of Exchange for G-Uni to use with low fee tier
  
## How to test 🧪
- Fork mainnet on HH after block 15774580
- Open G-Uni position
- Sense check Fee
